[Merged by Bors] - feat: Support arbitrary connectors using the generic connector (#436)
stackabletech/trino-operator: docs/modules/trino/pages/usage_guide/catalogs/index.adoc

= Catalogs

> Trino does not have a built-in catalog and instead `provides` connectors to external data sources.

Reviewer: the `provides` should not ticket but maybe cursive?

Author: Removed the quotes entirely

In this case the `hive` and `iceberg` catalogs will be used as they both match the `catalogLabelSelector` value `simple-trino`.

A `TrinoCluster` can, once created, detect and use new catalogs that have been subsequently created with a matching label. This also means that it is possible to reuse a `TrinoCatalog` within multiple `TrinoClusters`.

> === Generic fallback connector
> As Trino supports a lots of different connectors, we can't possible cover all the available connectors.
> In case the trino-operator does not support a specific connector we offer a fallback mechanism:

Reviewer: ```suggestion
=== Generic fallback connector

Trino supports lots of different connectors and we can not cover all the available connectors.
In case the Trino operator does not support a specific connector, there is a fallback mechanism:
```
